### ISSUE TITLE
RetroShare not using multiple source automatically

### ISSUE THREAD

**Issue by SebRut** [RetroShare/RetroShare#883]
I'm in a small network with some friends and when I start a download from shared files view retroshare only downloads it from that node even if multiple sources are available. If I start the download of this same file from multiple nodes then retroshare will combine them in one download. Is there a way to make retroshare automatically download from multiple sources without using the search view?

**Comment by felisucoibi** [comment]
Yes, rs should check in friend file lists avery hour or something like that to see if in friend lists the files are avaliable and add them as a source. I think there is some config to be enabled inside Network > left click on a friend > advancend (or something like that) > enable direct connections and the other option that don'0t remember name

**Comment by defnax** [comment]
@csoler close
